5_analyze_data/visualize_env.yml

```diff
@@ -24,4 +24,5 @@ dependencies:
   - conda-forge::r-reshape2
   - conda-forge::r-cowplot
   - conda-forge::r-patchwork
+  - conda-forge::r-pwr
```

Comment:
Consider specifying package versions (always)!

Reply:
As well, since @gwaybio originally made the PR for the `5_analyze_data` module and is more familiar with the packages, I refrain from adding any exact specific package versions. I am trying to get into the adding them for other environment now, so maybe I can `conda list` and add the versions but it might not be the ones that Greg has when he originally made it.